packages/client/package.json

```diff
@@ -34,6 +35,7 @@
     "@types/recursive-readdir": "2.2.0",
     "@types/tar-fs": "1.16.1",
     "jest-junit": "16.0.0",
+    "vitest": "2.0.1",
```

Comment:
Is this actually necessary? `vitest` is a dep in the root-level "package.json".

Also, looks like we're using a newer version at the root level `2.0.3`.